src/plat/qemu-arm-virt/config.cmake

```diff
@@ -60,7 +71,7 @@ if(KernelPlatformQEMUArmVirt)
     config_set(KernelARMPlatform ARM_PLAT qemu-arm-virt)
     set(DTBPath "${CMAKE_BINARY_DIR}/virt.dtb")
     set(DTSPath "${CMAKE_BINARY_DIR}/virt.dts")
-    if(KernelArmHypervisorSupport)
+    if(KernelArmHypervisorSupport OR KernelSel4ArchArmHyp)
```

Comment:
Why have both? Is there a difference in meaning between the two?

Reply:
`KernelArmHypervisorSupport` may not be set at this point as it's not declared until a later CMake file is processed.